docroot/sites/all/modules/features/build/midcamp_fields/midcamp_fields.features.field_base.inc

```diff
@@ -1445,6 +1575,31 @@ Prof.',
     'type' => 'list_boolean',
   );
 
+  // Exported field_base: 'field_ticket_id'.
+  $field_bases['field_ticket_id'] = array(
```

Comment:
Last year, the ticket field was a list with the key bring the ticket_class_id from eventbrite and the value being the ticket type name. The purpose is to be able to list the users purchased tickets on their profile page. It's also useful if a ticket was purchased for someone by someone else. In that case they would have the ticket id but no order id.

A callback to retrieve those values from Eventbrite would be pretty classy, but the tickets are not likely to change at this point. Last year they were manually put in by using the id and name values from the call https://www.eventbrite.com/developer/v3/endpoints/events/#ebapi-get-events-id-ticket-classes